web/README.md

```diff
@@ -112,9 +112,38 @@ run a demo locally with `npm run demo`, or [install the extension in your browse
 
 ### Testing
 
-To run all of the tests in this project, we currently require that you have [Chrome installed to its default location](https://www.google.com/chrome/).
+There are two parts of tests to this project:
+- Regular node tests, ran through `npm run test`. You must have built everything first as above. These have no special requirements.
+- Browser based tests, ran through `npm run wdio` with extra arguments as below. These take longer to run and require some setup.
 
-First, ensure you've build every package (see above), and then run `npm run test` to run the full suite of tests.
+## Browser based tests
+There are full integration tests that require a browser to run. We don't make any assumptions about your environment, and so you must specify it yourself.
+
+To run these tests, first build the project as above, then use `npm run wdio -- --arg1 --arg2` etc.
+
+### Local Browsers
+These are additive - you can specify multiple at the same time. You must have the given browsers installed locally though, or it will fail.
+
+- `--chrome` for Chrome
+- `--firefox` for Firefox
+- `--edge` for Edge
+
+### Browserstack (Mobile browsers)
+To run tests on mobile devices on Browserstack, pass the `--browserstack` argument.
+We also have our "minimum supported desktop browsers" available too, by additionally passing `--oldVersions`.
+
+You will need a Browserstack account (Maintainers may contact @Dinnerbone on Discord for an invite to the Ruffle team),
+and set the appropriate values to `BROWSERSTACK_USERNAME` and `BROWSERSTACK_ACCESS_KEY` environment variables.
```

Comment:
nits: Section names not in title case, and I think they call themselves "BrowserStack".